dspace-server-webapp/src/main/java/org/dspace/app/rest/BundleUploadBitstreamController.java

```diff
@@ -59,7 +59,7 @@
  * </pre>
  */
 @RestController
-@RequestMapping("/api/" + BundleRest.CATEGORY + "/" + BundleRest.PLURAL_NAME + "/"
```

Comment:
Why was this changed, perhaps a wrongful removal of the slash there ?

Reply:
Aside from this one issue everything looks good & works for me, things I tried:

* Build the old rest server & try a few requests
* Startup the server webapp & click around in the HAL Browser
* Login/logout in the HAL browser